doc/03-Configuration.md

```diff
@@ -0,0 +1,67 @@
+# Configuration
+
+![Icinga Notifications Web Preview](res/notifications-preview.png)
+
+If Icinga Web has been installed but not yet set up, please visit Icinga Web and follow the web-based setup wizard.
+For Icinga Web setups already running, log in to Icinga Web with a privileged user and follow the steps below to
+configure Icinga Notifications Web:
+
+<!-- {% if not icingaDocs %} -->
+
+## Module Activation
+
+If you just installed the module, do not forget to activate it on your Icinga Web instance(s) by using your
+preferred way:
+
+- Use Icinga Web's command-line interface on the webserver(s) and execute `icingacli module enable notifications`.
+- Visit Icinga Web, log in as a privileged user and activate the module under `Configuration →
+  Modules → Notifications` by switching the state from `disabled` to `enabled`.
+
+<!-- {% endif %} -->
+
+## Database Configuration
+
+Connection configuration for the database, which both,
+[Icinga Notifications](https://github.com/Icinga/icinga-notifications) and [Icinga Notifications Web](https://github.com/Icinga/icinga-notifications-web), use.
+
+!!! tip
+    If not already done, initialize your database by following the [instructions](https://icinga.com/docs/icinga-notifications/latest/doc/02-Installation#setting-up-the-database).
+
+1. Create a new resource for the Icinga Notifications database via the `Configuration → Application → Resources` menu.
+2. Configure the resource you just created as the database connection for the Icinga Notifications Web module using the
+   `Configuration → Modules → notifications → Database` menu.
+
+## Channels Configuration
+
+As the Icinga Notifications daemon notifies contacts in case of events and incidents, you need to configure appropriate 
+communication channels.
+
+The currently supported channels can be found [here](01-About.md#available-channels).
```

Comment:
I would remove this sentence as the link only leads back to the main page and confuses the user's walk-through without bringing any real additional benefit.

Reply:
The intention of my comment was from a novice user's point of view, following the manual while setting up Icinga Notifications. Speaking for myself, I would blindly follow this link and would be annoyed to find myself back at the _About_ page I have already read, being interrupted in the flow. This just might be me, but I would reduce the amount of links to other pages as much as possible to get a streamlined experience™.

Thus, just treat this as a comment regarding _my personal_ experience with manuals and howtos. If you disagree, just close this conversation thread. I won't be mad :)

Reply:
[Macros](https://jinja.palletsprojects.com/en/3.1.x/templates/#import) might be of benefit here.